webapp/src/components/simple/DISCOllaborative.vue

```diff
@@ -1,4 +1,6 @@
-<template><DISCO />llaborative</template>
+<template>
+  <DISCO /><span class="font-disco">llaborative</span>
```

Comment:
nitpicking: using `span` creates separated entities, so "disco" then "llaborative", instead of a single word. that does matter on screen readers and other accessibility tools.

```suggestion
  <span class="font-disco"><DISCO />llaborative</span>
```

Reply:
Oh good to know